src/generators/emoji/generator.ts

```diff
@@ -118,7 +119,7 @@ function replaceNumbers(str: string) {
 
 function removePunctuations(str: string): EmojiString {
   const lowerStr = str.toLowerCase();
-  if (!/[,|.|!|?|;|(|)|"]/g.test(str)) {
+  if (!/[,|.|!|?|;|(|)|"|'|:]/g.test(str)) {
```

Comment:
Could you please extract it to somewhere else, like a method or so, so it has a readable name? Writing regex is always easier than reading it, this would improve readability by a lot.

Reply:
done